meta/store.go

```diff
@@ -735,12 +734,20 @@ func (s *Store) serveExecListener() {
 		// Accept next TCP connection.
 		var err error
 		conn, err := s.ExecListener.Accept()
-		if err != nil {
-			if strings.Contains(err.Error(), "connection closed") {
+		if opErr, ok := err.(*net.OpError); ok && opErr.Temporary() {
```

Comment:
This doesn't look right to me.

If the error is not temporary, then simply return. Otherwise continue in the loop. Won't that work?

Also, why are you sleeping for a second? Is that debug code?